.coderabbit.yaml

```
@@ -0,0 +1,45 @@
+    - '!**/generated/**'
+
+  path_instructions:
+    - path: 'src/**/*.kt'
+      instructions: |
+        ## 1. Kotlin 코딩 컨벤션
+        - Kotlin 공식 코딩 컨벤션을 준수해야 합니다.
+        - 클래스는 PascalCase, 함수/변수는 camelCase, 상수는 UPPER_SNAKE_CASE로 작성하세요.
+        - 변경 불가능한 데이터는 val을, 변경이 필요한 경우에만 var를 사용하세요.
+        - Null Pointer Exception을 유발하는 !! 연산자 사용을 금지합니다.
+        - 의미를 알 수 없는 숫자나 문자열(매직 넘버) 대신 명명된 상수를 사용하세요.
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)